### PR TITLE
Codeception test setup

### PR DESCRIPTION
**Pull request by IvanGrginovInf**
Done
---
* Setup WP Browser
* Setup an example unit test
* Added unit tests to workflow

### CODE REVIEW

**Review by iruzevic — APPROVED** [review]

File: .env.testing [hunk 1]
```diff
@@ -0,0 +1,18 @@
+TEST_SITE_DB_DSN=mysql:host=localhost;dbname=test
```
Comment (dingo-d) [inline .env.testing:1]: I'm 99% sure this won't work with the integration test setup on GH actions, but we'll cross that bridge when we get there 😄

Comment (IvanGrginovInf) [reply .env.testing:1]: Yeah, it won't unfortunately. Just a placeholder as setup as default by `wpbrowser`.

File: tests/unit/Helpers/ComponentsTest.php [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+{
+    /**
+     * @var \UnitTester
+     */
+    protected $tester;
+
+    public function testEnsureString()
+    {
+        $this->assertIsString(Components::ensureString('asdad'));
+        $this->assertIsString(Components::ensureString(['aaa', 'bbb']));
```
Comment (dingo-d) [inline tests/unit/Helpers/ComponentsTest.php:15]: Try to test the 'unhappy path' as well. Like, what would happen if you pass some totally bonkers thing to `Components::ensureString`? Will it throw an error, or will try to turn that into a string, but return false? Will it completely break the app?

**Review by dingo-d — CHANGES_REQUESTED** [review]
One small comment for the test. Good work!

File: tests/unit/Helpers/ComponentsTest.php (hunk 2 above)
Comment (IvanGrginovInf) [reply tests/unit/Helpers/ComponentsTest.php:15]: Found a bug in my function because of this comment, nice! 😄

**Pull request closed without merging**